src/runtime/pkg/direct-volume/utils.go

```diff
@@ -0,0 +1,95 @@
+package volume
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	mountInfoFileName        = "mountInfo.json"
+	defaultTimeout = 3 * time.Second
+)
+var kataDirectVolumeRootPath = "/run/kata-containers/shared/direct-volumes"
+
+// MountInfo contains the information for the Kata agent to mount a filesystem on the volume inside the guest VM.
```

Comment:
Some info is consumed by the shim, and other info is eventually passed to the guest agent. What do you think about rewording, like:
```
MountInfo contains the information needed by Kata to consume a host block device and mount it as a filesystem inside the guest VM.
```